.github/codeql/codeql-config.yml

```diff
@@ -5,3 +5,4 @@ paths-ignore:
   - 'tools/floodgate/**'
   - 'tools/translation/**'
   - node_modules
+  - libs/deps/mas
```

Comment:
Why is this ignored? Is it enabled on the MAS side? I'd argue that anything that goes into Milo should follow its standards. If the larger community doesn't agree, then we should consider disabling it for all dependencies, not just this one.

Reply:
MAS had some issues regarding lit creating code that's being flagged, which can't easily be changed / updated

Reply:
see here https://github.com/adobecom/milo/pull/2690#discussion_r1711189410 @overmyheadandbody